### ISSUE TITLE
Can we add string.include method?

### ISSUE THREAD

**Issue by hk-skit** [olivernn/augment.js#19]
As described here [Mozila doc](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/String/includes), there's a method string.includes to check whether a string contains searchString or not. 

```

if (!String.prototype.includes) {
  String.prototype.includes = function(search, start) {
    'use strict';
    if (typeof start !== 'number') {
      start = 0;
    }

    if (start + search.length > this.length) {
      return false;
    } else {
      return this.indexOf(search, start) !== -1;
    }
  };
}

```

We should also add this in augment.js

**Comment by plugn** [comment]
It is possible to use String.prototype.indexOf() instead:
[str.indexOf(searchValue[, fromIndex]) ](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/indexOf)
with the same effect

**Comment by hk-skit** [comment]
Yeah we can achieve the same with it but the thing is that every time we need to check whether the index is not equals -1.
